apps/fz_http/lib/fz_http_web/endpoint.ex

```diff
@@ -29,6 +29,13 @@ defmodule FzHttpWeb.Endpoint do
     ],
     longpoll: false
 
+  socket "/gateway", FzHttpWeb.GatewaySocket,
+    websocket: [
+      check_origin: false,
+      error_handler: {FzHttpWeb.GatewayErrorHandler, :handle_error, []}
```

Comment:
So there are a few things here:
1. I think we should use the `User-Agent` connection header to pass the gateway version to the panel, so that we can render it in the UI.
2. This header will be also useful in browser tests because we will user the user agent to pass 
`Ecto.SQL.Sandbox` metadata.
3. We also probably want to assign a gateway IP to the `socket`. I even think that we might want to use it as default gateway name (it should uniquely identify the gateway as long as it's assigned IP is the same)
```suggestion
    websocket: [
      connect_info: [
        :user_agent,
        :peer_data,
      ],
      check_origin: false,
      error_handler: {FzHttpWeb.GatewayErrorHandler, :handle_error, []}
```

Reply:
But where would we user `user_agent` in the gateway? I think if it's just for the logs we don't need to pass it here? And for the same reason until we don't settle on using (or not) the ip for the gateway name we shouldn't add it here.